src/timer/CountDownTimer.test.jsx

```diff
@@ -293,4 +293,58 @@ describe('ExamTimerBlock', () => {
 
     await waitFor(() => expect(screen.getByText('00:00:19')).toBeInTheDocument());
   });
+
+  const timesToTest = {
+    // Because times are rounded down, these values are 60 seconds off
+    '2 hours and 29 minutes': 9000,
+    '1 hour and 29 minutes': 5400,
+    '2 hours and 1 minute': 7320,
+    '1 hour and 1 minute': 3720,
+    '2 hours and 0 minutes': 7260,
+    '1 hour and 0 minutes': 3660,
+    '29 minutes': 1800,
+  };
+  Object.keys(timesToTest).forEach((time) => {
+    it('Accessibility time string appears as expected based on time in seconds remaining', async () => {
```

Comment:
```suggestion
    it( `Accessibility time string appears as expected based on ${timesToTest[time]} seconds remaining`, async () => {
```
small suggestion, this is going to output a bunch of tests with the exact same name "ExamTimerBlock > Accessibility time string appears as expected based on time in seconds remaining" so if one fails you can't tell what test it is. I think you could add the `time` variable into the name of the test.